girder/api/describe.py

```diff
@@ -436,7 +436,7 @@ def __init__(self, templatePath=None):
         self.vars = {
             'apiRoot': '',
             'staticRoot': '',
-            'title': 'Girder - REST API Documentation',
+            'brandName': ModelImporter.model('setting').get(constants.SettingKey.BRAND_NAME),
```

Comment:
I added a event Handler on the Api page but it's not working when I try to set the brandname with a blank (set by default)